c/httpfileservice.c

```diff
@@ -401,10 +452,17 @@ static int copyUnixDirectory(char *oldAbsolutePath, char *newAbsolutePath, int f
   }
   
   status = fileInfo(newAbsolutePath, &info, &returnCode, &reasonCode);
-  if (status == 0 && !forceCopy) {
+  if (status == 0) {
     zowelog(NULL, LOG_COMP_RESTFILE, ZOWE_LOG_DEBUG,
-            "Directory already exists %s\n", newAbsolutePath);    
-    return RC_HTTP_FILE_SERVICE_ALREADY_EXISTS;  
+            "Directory already exists %s\n", newAbsolutePath); 
+
+    if (!forceCopy) {          
+      return RC_HTTP_FILE_SERVICE_ALREADY_EXISTS;  
+    }
+    else if (forceCopy && deleteUnixDirectory(newAbsolutePath))
+    {
+      return RC_HTTP_FILE_SERVICE_UNDEFINED_ERROR;
```

Comment:
The error code has to specific, e.g. `RC_HTTP_FILE_SERVICE_FAILED_TO_CLEANUP_TARGET_DIR`.

Reply:
Fixed it